lang/src/main/antlr/PartiQL.g4

```diff
@@ -76,15 +81,38 @@ ddl
     ;
 
 createCommand
-    : CREATE TABLE symbolPrimitive                                                              # CreateTable
+    : CREATE TABLE tableName PAREN_LEFT tableDef PAREN_RIGHT                                    # CreateTable
     | CREATE INDEX ON symbolPrimitive PAREN_LEFT pathSimple ( COMMA pathSimple )* PAREN_RIGHT   # CreateIndex
     ;
 
 dropCommand
-    : DROP TABLE target=symbolPrimitive                         # DropTable
+    : DROP TABLE target=tableName                               # DropTable
     | DROP INDEX target=symbolPrimitive ON on=symbolPrimitive   # DropIndex
     ;
 
+tableDef
+    : tableDefPart ( COMMA tableDefPart )*
+    ;
+
+tableDefPart
+    : columnName type columnConstraint*                             # ColumnDeclaration
+    | ( CONSTRAINT tableConstraintName )?  tableConstraintDef       # TableConstraint
+    ;
+
+columnConstraint
+    : ( CONSTRAINT columnConstraintName )?  columnConstraintDef
+    ;
+
+columnConstraintDef
+    : NOT NULL                                  # ColConstrNotNull
+    | NULL                                      # ColConstrNull
+    | CHECK PAREN_LEFT expr PAREN_RIGHT         # ColConstrCheck
```

Comment:
Maybe it makes sense going only w/ `NOT NULL` and perhaps `NOT MISSING` for now until we flesh out the semantics of the rest in PartiQL (like `CHECK`).

Reply:
Evaluation can also be evaluation of a `DML` statement like `INSERT`. In such cases `NOT MISSING` can mean that the attribute is non-optional and the incoming data must provide a value for such attribute. Having said that it makes sense to leave it out until we flesh out the `SchemaType` semantics.